.env

```diff
@@ -1,4 +1,4 @@
-REACT_APP_RINKEBY_CONTRACT=0xd85DFAB4526ca8E36b49c98896A34cFD0B995Ba0
+REACT_APP_CLONE_ABI=[ { "anonymous": false, "inputs": [ { "indexed": false, "internalType": "address", "name": "account", "type": "address" }, { "indexed": false, "internalType": "uint256", "name": "shares", "type": "uint256" } ], "name": "PayeeAdded", "type": "event" }, { "anonymous": false, "inputs": [ { "indexed": false, "internalType": "address", "name": "from", "type": "address" }, { "indexed": false, "internalType": "uint256", "name": "amount", "type": "uint256" } ], "name": "PaymentReceived", "type": "event" }, { "anonymous": false, "inputs": [ { "indexed": false, "internalType": "address", "name": "to", "type": "address" }, { "indexed": false, "internalType": "uint256", "name": "amount", "type": "uint256" } ], "name": "PaymentReleased", "type": "event" }, { "inputs": [ { "internalType": "uint256", "name": "index", "type": "uint256" } ], "name": "payee", "outputs": [ { "internalType": "address", "name": "", "type": "address" } ], "stateMutability": "view", "type": "function" }, { "inputs": [ { "internalType": "address payable", "name": "account", "type": "address" } ], "name": "release", "outputs": [], "stateMutability": "nonpayable", "type": "function" }, { "inputs": [ { "internalType": "address", "name": "account", "type": "address" } ], "name": "released", "outputs": [ { "internalType": "uint256", "name": "", "type": "uint256" } ], "stateMutability": "view", "type": "function" }, { "inputs": [ { "internalType": "address", "name": "account", "type": "address" } ], "name": "shares", "outputs": [ { "internalType": "uint256", "name": "", "type": "uint256" } ], "stateMutability": "view", "type": "function" }, { "inputs": [], "name": "totalReleased", "outputs": [ { "internalType": "uint256", "name": "", "type": "uint256" } ], "stateMutability": "view", "type": "function" }, { "inputs": [], "name": "totalShares", "outputs": [ { "internalType": "uint256", "name": "", "type": "uint256" } ], "stateMutability": "view", "type": "function" }, { "stateMutability": "payable", "type": "receive" }, { "inputs": [ { "internalType": "address[]", "name": "_payees", "type": "address[]" }, { "internalType": "uint256[]", "name": "_shares", "type": "uint256[]" } ], "name": "initialize", "outputs": [], "stateMutability": "payable", "type": "function" } ]
```

Comment:
@th3o6a1d we don't need to have these ABI in ENVs as they don't depending on the environment. we can move them to typescript file.
Also, as @cbrzn recommended, let convert these ABI to [Human-Readable](https://docs.ethers.io/v5/api/utils/abi/formats/#abi-formats--human-readable-abi). you can follow [this guide](https://docs.ethers.io/v5/api/utils/abi/formats/#abi-formats--converting-between-formats) to make the transformation.

Reply:
FYI, one alternative way of consuming ABIs is using [TypeChain](https://github.com/ethereum-ts/Typechain).

Reply:
Good call. Removed these ABIs from the `.env` file. Now it's just the rinkeby contract location.